check_rpmspec_collection.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [ "$1" = "-h" -o "$1" = "--help" ]; then
+    echo "Description - Verify rpm package count with various combination of ansible and collection_artifact options."
+    echo "            - Check there is no README.html files in /usr/share/andible/collections."
+    echo "Usage: $0 [ -h | --help ] | [ fedpkg | rhpkg [ branch_name ] ]"
+    exit 0
+fi
+
+if [ ! -f *.spec ]; then
```

Comment:
if there is more than one spec file you will get an error like this:
```
bash: [: another.spec: binary operator expected
```

Reply:
Maybe something like this:
```bash
if ls -1 *.spec | wc -l | grep -q '^1$'; then
  echo good
else
  echo there must exactly 1 spec file in $(pwd)
  exit 1
fi
```
if there are no spec files the output looks like this:
```
ls: cannot access '*.spec': No such file or directory
there must exactly 1 spec file in /home/rmeggins/fedpkg/linux-system-roles
```

Reply:
> if there is more than one spec file you will get an error like this:
> 
> ```
> bash: [: another.spec: binary operator expected
> ```

Thanks for the solution. I was wondering if there were such case having multiple spec files (and hoping there isn't... :). But, there could be?

Reply:
> But, there could be?

There should only ever be one file matching `*.spec` - so if you want to keep `if [ -f *.spec ]` that's fine - it should be pretty obvious to a developer what the problem is.